tests/framework/Samplers/DynamicEventTrees/gold/DETshort/output_collapsed/testprint_hist_collapsed.csv

```diff
@@ -0,0 +1,29 @@
+dummy_for_branch,ProbabilityWeight,RAVEN_isEnding,PointProbability,ProbabilityWeight-<distribution>zeroToOne,RAVEN_parentID,prefix,filename
+18.02,0.0,1.0,0.0,0.8,DET_test_1-1-2,DET_test_1-1-2-1,/Users/alfoa/projects/raven_github/raven/tests/framework/Samplers/DynamicEventTrees/DETshort/output_collapsed/testprint_hist_collapsed_15.csv
```

Comment:
I don''t think things with "/Users/alfoa" are going to translate well into tests.